systemtests/environment.in

```diff
@@ -56,3 +56,21 @@ export db_password=@db_password@
 export db_user=@db_user@
 
 export PAM_WRAPPER_LIBRARIES=@PAM_WRAPPER_LIBRARIES@
+
+#
+# Selenium test
+#
+
+export BAREOS_WEBUI_CONFDIR=@WEBUICONFDIR@
+
+export BAREOS_BROWSER=chrome
+export BAREOS_CHROMEDRIVER_PATH=@CHROMEDRIVER@
+
+export BAREOS_BASE_URL=http://localhost:@php_port@/
+export BAREOS_USERNAME=admin
+export BAREOS_PASSWORD=admin
+export BAREOS_CLIENT_NAME=bareos-fd
+export BAREOS_RESTOREFILE=${PROJECT_BINARY_DIR}
+export BAREOS_LOG_PATH=${logdir}
+export BAREOS_DELAY=1
```

Comment:
```suggestion
export BAREOS_WEBUI_DELAY=1
```
This change would avoid name clashes or confusions in the future.

Reply:
I agree, to avoid such clashes let's define environment vars introduced for webui have to have the prefix BAREOS_WEBUI_ .